src/main/java/com/arcbees/website/client/BootstrapperImpl.java

```diff
@@ -52,9 +55,16 @@ public void onBootstrap() {
         analytics.enablePlugin(AnalyticsPlugin.DISPLAY);
         analytics.enablePlugin(AnalyticsPlugin.ENHANCED_LINK_ATTRIBUTION);
 
+        int variationId = getVariationId();
+        experimentHolder.setVariationId(variationId);
+
         validateNameTokenLanguageAndRevealPlace();
     }
 
+    private native int getVariationId() /*-{
```

Comment:
In fact this should be in UA library. At the moment this is here for to ensure it's working. Once we confirm this works I'll extract this more cleanly to UA library